### ISSUE TITLE
Cannot figure out the error from the error-message

### ISSUE THREAD

**Issue by SupunS** [ballerina-platform/ballerina-lang#17750]
**Description:**
```ballerina
type OrderInfo record {|
    string id;
    string name;
    string description;
    
|};

@http:ServiceConfig {
    basePath: "/PizzaService"
}
service PizzaService on new http:Listener(9090) {
    @http:ResourceConfig {
        methods: ["POST"],
        path: "/order",
        consumes: ["text/json", "application/json"]
    }
    resource function orderPizza(http:Caller caller, http:Request request, OrderInfo pizzaOrder) {
        var x = caller -> respond("ordered!");
    }
}
```

Following error is shown when compiling the above service:
```
error: supun/http2grpc:0.1.0::proxy.bal:10:5: Invalid parameter(s) in the resource signature
```

However, it does not give any clue about the actual issue, neither how to resolve it.

**Comment by chamil321** [comment]
Fixed in 1.0.0-beta
Following error is shown at compile time when excess resource signature params are defined
```
error: .::main.bal:14:5: invalid resource parameter(s): cannot specify > 2 parameters without specifying path config and/or body config in the resource annotation
```
Hence closing the PR

**Comment by M-Keidler-GROB** [comment]
I updated to 1.0.0 (stable) and it throws that error:
`error: .::cybusAPI.bal:33:5: invalid resource parameter(s): cannot specify > 2 parameters without specifying path config and/or body config in the resource annotation`

code snippet:
`@http:ServiceConfig {
    basePath: "/cybus"
}

service cybus on new http:Listener(9090) {

    @http:ResourceConfig {
        methods: ["GET"],
        path: "/appliances"
    }
    resource function getListOfAppliances(http:Caller caller, http:Request request) {
        
        http:Client clientEndpoint = new("https://portal.cybus.io");
        
        logInfo("GET request: Names of Appliances");
                
        http:Request req = new;
               
        json payload = { "query": "query { appliances { name, state, id } }" };
        req.setJsonPayload(payload, contentType = "application/json");
        req.setHeader("Authorization", "Bearer " + getToken());
       
        var response = clientEndpoint->post("/graphql", req);

        handleResponse(response, caller);                           
    }

    @http:ResourceConfig {
        methods: ["POST"],
        path: "/appliances"
    }
    resource function createNewAppliance(http:Caller caller, http:Request request, json tennantID) {
        `
